package.json

```diff
@@ -23,7 +23,8 @@
   },
   "scripts": {
     "bootstrap": "npm i && lerna bootstrap",
-    "release": "lerna publish --cd-version=prerelease --preid=alpha",
+    "build:full": "npm run clean && npm run build && npm run mocha && npm run lint",
```

Comment:
Please move `build:full` closer to other scripts starting with `build` prefix. To be honest, `build:full` does not make sense to me in the context of existing scripts `build` and `build:current`. I think a better name would be something like `clean-build-test` or `build-and-test`.